fmf/utils.py

```diff
@@ -565,6 +569,32 @@ def set_cache_directory(cache_directory):
     global _CACHE_DIRECTORY
     _CACHE_DIRECTORY = cache_directory
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Get Whole Tree from remote
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+def get_remote_tree(url, ref, path):
```

Comment:
I'm not sure about default values - I'd prefer to have just one place where they belong - fmf.utils.fetch in this case.